Makefile.dep

```diff
@@ -98,6 +98,7 @@ ifneq (,$(filter gnrc_dhcpv6_client,$(USEMODULE)))
 endif
 
 ifneq (,$(filter gnrc_uhcpc,$(USEMODULE)))
+  DEFAULT_MODULE += auto_init_gnrc_uhcpc
```

Comment:
Instead of pulling this by default here why not rather add the following later in the file:

```mk
ifneq (,$(filter auto_init,$(USEMODULE)))
  ifneq (,$(filter gnrc_uhcpc,$(USEMODULE)))
    DEFAULT_MODULE += auto_init_gnrc_uhcpc
  endif

  [... other auto_init_% default modules] 
endif
```

?

Reply:
Looks good, I'll do it later, for now I wan't to see if if all tests pass. If people like it I will go into formating